### DISCUSSION TITLE
Wiki file updater for two artifacts (Azure wiki, pipelines)

### DISCUSSION THREAD

**Discussion by kahaboom** [rfennell/AzurePipelines#1413] [Q&A] [answered]
Hello dear @rfennell and community. 

Situation : I have two build pipelines. For backend and frontend. The file which is creating after is called by "build number" of the build pipeline from the backend. 

Is it possible to use both (frontend and backend) artifacts for one release notes generator task and if the files for back and front have the same name, then merge them into one file ? 

For example, backend and frontend have new release and the file will be called 1.1.1.md. Not to create two files with the same name, can I automatically merge them into one ?

**Comment by rfennell** [comment, marked as answer]
You would have to have each pipeline generate their own release notes markdown file, and then you write a script to merge them. Not a particularly pleasant thing to write.

What we do is have each pipeline generate their markdown file, then upload it to a WIKI. We can then generate a PDF off this WIKI. [See video](https://www.youtube.com/watch?v=xaV3dFoQdV8&t=332s)

The other option is to have a single pipeline that builds both halves of the product

**Reply by kahaboom** [reply]
Thank you @rfennell
Will think something about it :)
